### PR TITLE
Adjust notify service for icc

### PR DESCRIPTION
**Pull request by tsiegleauq**
Adjust notify service for icc

Enable notify via new ICC integration.
Adjust previously used connection manager to use
new communiction manager

Attach osListenEditing to motion detail to get ICC notifications for
edits

### CODE REVIEW

**Comment by tsiegleauq** [comment]
@FinnStutzenstein please review later

File: client/src/app/core/core-services/http.service.ts [hunk 1]
```diff
@@ -236,4 +236,29 @@ export class HttpService {
         const file = await this.get<Blob>(url, {}, {}, headers, 'blob');
         return await toBase64(file);
     }
+
+    /**
+     * Helper function to check if a given URL was reachable
+     * @param url
+     */
+    public async isServiceOnline(url: string): Promise<boolean> {
```
Comment (FinnStutzenstein) [inline client/src/app/core/core-services/http.service.ts:244]: Is this needed? I do not see any reference.

File: client/src/app/core/core-services/notify.service.ts [hunk 2]
```diff
@@ -97,19 +101,81 @@ export class NotifyService {
...
+
+    private async connect(meetingId: number): Promise<void> {
+        if (!meetingId) {
+            throw new Error('Cannot connect to ICC, no meeting ID was provided');
+        }
+
+        this.disconnect();
+
+        const iccMeeting = `${IccPath}?meeting_id=${meetingId}`;
+        this.commManager.registerEndpoint('icc', iccMeeting, iccHealthPath, HTTPMethod.GET);
```
Comment (FinnStutzenstein) [inline client/src/app/core/core-services/notify.service.ts:143]: Nice, that this worked so easy :)

File: client/src/app/core/core-services/notify.service.ts [hunk 3]
```diff
@@ -180,7 +247,8 @@ export class NotifyService {
         if (channels) {
             notify.to_channels = channels;
         }
-        this.websocketService.send('notify', notify);*/
+
+        await this.httpService.post<unknown>('/system/icc/send', notify);
```
Comment (FinnStutzenstein) [inline client/src/app/core/core-services/notify.service.ts:251]: You can use `IccPath` from above.

**Pull request merged**